samples/java-eventsourced-counter/src/main/java/com/example/domain/CounterJournalToTopicActionTestKit.java

```diff
@@ -0,0 +1,73 @@
+package com.example.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+import com.akkaserverless.javasdk.action.Action;
+import com.akkaserverless.javasdk.action.ActionCreationContext;
+import com.akkaserverless.javasdk.testkit.EventSourcedResult;
+import com.akkaserverless.javasdk.testkit.ActionResult;
+import com.akkaserverless.javasdk.testkit.impl.ActionResultImpl;
+import com.akkaserverless.javasdk.impl.action.ActionEffectImpl;
+import com.example.actions.CounterJournalToTopicAction;
+import com.akkaserverless.javasdk.testkit.impl.StubActionCreationContext;
+import com.akkaserverless.javasdk.testkit.impl.StubActionContext;
+import com.example.actions.CounterTopicApi;
+import java.util.Optional;
+
+/**
+ * TestKit for unit testing Counter
+ */
+public final class CounterJournalToTopicActionTestKit {
+
+  private CounterJournalToTopicAction action;
+  private List<Object> effects = new ArrayList<Object>();
```

Comment:
As the actions commands are completely independent and stateless I don't think it makes sense to keep track of them like this, just return the effect to the user for asserting on each call and that's it.

Reply:
done